packages/loopring_v3/test/testExchangeBlocks.ts

```diff
@@ -419,16 +485,28 @@ contract("Exchange", (accounts: string[]) => {
           await exchangeTestUtil.submitPendingBlocks(
             (onchainBlocks: OnchainBlock[], blocks: Block[]) => {
               assert(blocks.length === 1, "unexpected number of blocks");
-                const auxiliaryData: any[] = [];
-                for (const [i, tx] of blocks[0].internalBlock.transactions.entries()) {
-                  if (tx.txType === "Deposit" || (tx.txType === "Transfer" && tx.type > 0)) {
-                    auxiliaryData.push([i, web3.utils.hexToBytes("0x")]);
-                  } else if (tx.txType === "AccountUpdate" && tx.type > 0) {
-                    auxiliaryData.push([i, web3.utils.hexToBytes(tx.onchainSignature)]);
-                  }
+              const auxiliaryData: any[] = [];
+              for (const [
+                i,
+                tx
+              ] of blocks[0].internalBlock.transactions.entries()) {
+                if (
+                  tx.txType === "Deposit" ||
+                  (tx.txType === "Transfer" && tx.type > 0)
+                ) {
+                  auxiliaryData.push([i, web3.utils.hexToBytes("0x")]);
+                } else if (tx.txType === "AccountUpdate" && tx.type > 0) {
+                  auxiliaryData.push([
+                    i,
+                    web3.utils.hexToBytes(tx.onchainSignature)
+                  ]);
+                }
```

Comment:
Sometimes I really dislike this auto formatting making things much less readable...

Reply:
I think you can change the eslint config such that it can reformat it into something you like:)